include/re_rtp.h

```diff
@@ -311,3 +311,40 @@ static inline int16_t rtp_seq_diff(uint16_t x, uint16_t y)
 {
 	return (int16_t)(y - x);
 }
+
+
+/** NTP Time */
+struct ntp_time {
```

Comment:
I wonder if we should rename this struct?

Reply:
no problem.

Do you mean `struct rtp_ntp_time` or something else ?